r/R/dplyr-eval.R

```diff
@@ -20,6 +20,8 @@ arrow_eval <- function(expr, mask) {
   # with references to Arrays (if .data is Table/RecordBatch) or Fields (if
   # .data is a Dataset).
 
+  add_user_functions_to_mask(expr, mask)
```

Comment:
It might be nice to have a comment here explaining what this is — it's right below on line 54, so maybe that's enough, but something quick and descriptive (mostly that this is _not_ UDFs and instead about R funcs in the parent/global environment

Reply:
And here it might be nice to describe why: we do this because some functions can be reduced to functions that do have arrow bindings without needing to register one yourself or the like

Reply:
Revised, PTAL

Reply:
That's fantastic, thanks